skimage/_shared/fast_exp.h

```diff
@@ -3,15 +3,18 @@
  * Reference [2]: https://doi.org/10.1162/089976600300015033
  * Additional improvements by Leonid Bloch. */
 
+#include <stdint.h>
+
 /* use just EXP_A = 1512775 for integer version, to avoid FP calculations */
-#define EXP_A (1512775.3951951856938)  /* 2^20*ln2 */
+#define EXP_A (1512775.3951951856938)  /* 2^20÷ln2 */
 /* For min. RMS error */
 #define EXP_BC 1072632447              /* 1023*2^20 - 60801 */
 /* For min. max. relative error */
 /* #define EXP_BC 1072647449 */        /* 1023*2^20 - 45799 */
 /* For min. mean relative error */
 /* #define EXP_BC 1072625005 */        /* 1023*2^20 - 68243 */
 
+
```

Comment:
Why an extra line? In two places below as well... In any case, it has nothing to do with introducing a single precision version. :)

Reply:
One of my PEP8 habits... Not relevant here, you are right.